### ISSUE TITLE
#{mouse_word} / #{mouse_x} / #{mouse_y} not populated for status bar clicks?

### ISSUE THREAD

**Issue by scottgasch** [tmux/tmux#2913]
### Issue description

When I set up a mouse action on status bar clicks I find that the #{mouse_word} is blank:

bind -n MouseDown1StatusLeft display "#{mouse_word}"

Same with #{mouse_x} and #{mouse_y}.

I'd love to be able to change the action that is taken upon mouse click to be sensitive to what part of the left / right status bar I clicked on instead of only being able to do one thing per click.  Is there a way to populate these on all mouse events and/or a way to achieve what I'm trying to do here?

Please provide the following information:

* tmux version (`tmux -V`).

3.2a

* Platform (`uname -sp`).

FreeBSD amd64

* $TERM inside and outside of tmux (`echo $TERM`).

xterm-256color

**Comment by nicm** [comment]
`mouse_x` and `mouse_y` are a position inside a pane so they don't appear on the status line, although there is no real reason they shouldn't. Try this please: [x.diff.txt](https://github.com/tmux/tmux/files/7286231/x.diff.txt)

**Comment by scottgasch** [comment]
No dice.  It applies and builds cleanly but when I do this:

bind -n MouseDown1StatusLeft display -d 1000 "word: #{mouse_word} x:
#{mouse_x} y: #{mouse_y}"

I get this:

word:  x:  y:

Thanks for the pointer at the relevant code.

**Comment by nicm** [comment]
Works for me, show server log please.

**Comment by scottgasch** [comment]
I tried to attach the server log but it's too large for gmail.  I don't
know if my previous message made it through w/o the attachment or not but
you're right, it works:

Sorry, you're right.  I patched and built and killed my session but the
server process itself was seemingly running the old binary.  I killed it /
restarted the server and I'm getting a mouse_x now.  Including the server
logs in case they are useful.

Thank you, I'll use this to do more interesting things based on where I
click on the status bar left/right.

Scott
